components/function-controller/internal/controllers/serverless/gitops.go

```diff
@@ -3,11 +3,21 @@ package serverless
 import (
 	"fmt"
 
-	ctrl "sigs.k8s.io/controller-runtime"
-
 	"github.com/kyma-project/kyma/components/function-controller/internal/git"
+	serverlessv1alpha2 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha2"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+//TODO refactor - move to system state
```

Comment:
Do you plan to work on it in this story or the next one?

Reply:
This is a leftover, will delete it.